modules/storage-persistent-storage-pv.adoc

```diff
@@ -136,26 +136,24 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 endif::[]
 |===
 --
-1. ReadWriteOncePod access mode for persistent volumes is a Technology Preview feature.
+1. RWOP uses the SELinux mount feature. Supported vendors include ODF, AWS EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere.
```

Comment:
RWOP uses the SELinux **context** mount feature
Also the vendor list only includes drivers we ship, we should add a sentense for third party drivers.

`This feature is driver dependent and enabled by default in ODF, AWS EBS, Azure Disk, GCP PD, IBM VPC Block, Cinder, and vSphere. For third party drivers please contact your storage vendor.`

Reply:
Done.